keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/ConstraintModelConverter.kt

```diff
@@ -11,14 +12,14 @@ import org.springframework.stereotype.Component
  */
 @Component
 class ConstraintModelConverter(
-  evaluators: List<ConstraintEvaluator<*>>
+  private val extensionRegistry: ExtensionRegistry
 ) : SubtypesModelConverter<Constraint>(Constraint::class.java) {
 
-  override val subTypes = evaluators.map { it.supportedType.type }
+  override val subTypes: List<Class<out Constraint>>
+    get() = mapping.values.toList()
 
   override val discriminator: String? = Constraint::type.name
 
-  override val mapping: Map<String, Class<out Constraint>> = evaluators.associate {
-    it.supportedType.name to it.supportedType.type
-  }
+  override val mapping: Map<String, Class<out Constraint>>
+    get() = extensionRegistry.extensionsOf()
 }
```

Comment:
I'd like to unify all these implementations as they're basically all the same. I'll probably do another commit to this PR.